app/poros/user.rb

```diff
@@ -0,0 +1,9 @@
+class User
+  attr_reader :id, :name
+
+  def initialize(data)
+    binding.pry
```

Comment:
Another stray pry.  We should make sure we try not to push these up when we want to make a pull request.